_posts/2020-02-27-consumption-happiness.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Reducing Consumption while Increasing Happiness"
+date:   2018-11-25 22:40:05
+categories: jekyll update
+tags: book-review
+---
+
+"It's time to stop pointing fingers, finger pointing gets us no-where, Steve!"
+~ Randy Marsh, South Park
+
+![carbon-footprint](/assets/carbon-footprint.png)
```

Comment:
I like the infographic presented up front. It complements the follow up paragraph.